### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by cgeorgilakis**

### CODE REVIEW

**Comment by cgeorgilakis** [comment]
Added unreleased changes.
Maybe changelog need sychronization before merged.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,40 @@ For Keycloak upstream changelog please see https://www.keycloak.org/docs/latest/
   Full Keycloak upstream jira issue can be shown if filtered by Fix version. For example [Keycloak jira issue for 15.0.2 version](https://issues.redhat.com/browse/KEYCLOAK-19161?jql=project%20%3D%20keycloak%20and%20fixVersion%20%3D%2015.0.2)
```
Comment (NicolasLiampotis) [inline CHANGELOG.md:8]: Replace introduction with:
```
# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,9 +8,40 @@ For Keycloak upstream changelog please see https://www.keycloak.org/docs/latest/
...
+
 ### Added
+- Expand invite group member search to include group admins [RCIAM-1343](https://jira.argo.grnet.gr/browse/RCIAM-1343)
+- Requires Approval - Create Enrollment Page [RCIAM-1345](https://jira.argo.grnet.gr/browse/RCIAM-1345)
 
 ### Changed
+- Improve group member/ admin email [Improve group member/ admin email invitation](https://trello.com/c/uDaQ5usH/2581-improve-group-member-admin-email-invitation)
+
+### Fixed
+- Correct notifications for submitting enrollment rrequest
```
Comment (NicolasLiampotis) [inline CHANGELOG.md:20]: Fix typo rrequest --> request

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,9 +8,40 @@ For Keycloak upstream changelog please see https://www.keycloak.org/docs/latest/
...
+
+## [0.14.0] - 2024-04-08
+
+### Added
+
+- Support advance group management entity
```
Comment (NicolasLiampotis) [inline CHANGELOG.md:26]: Fix typo advance --> advanced

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,9 +8,40 @@ For Keycloak upstream changelog please see https://www.keycloak.org/docs/latest/
...
+- Support for enrollement requests to join in a group
+- Support for invitation to join in a group
+- Support for invitation to become group admin
+- Group admin can add another group admin
+- Suspend and reactivate group member
+- Group membership expiration
+- Extra support for account events
+- Group member being able to leave group
+- Support for pending group members
+- Support for eduperson_entitlment user attribute based on group memberships
```
Comment (NicolasLiampotis) [inline CHANGELOG.md:39]: Support for eduperson_entitlment user attribute based on group memberships --> Support for [AARC-G069](https://aarc-community.org/guidelines/aarc-g069) compatible entitlement user attribute URN value based on group membership

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (cgeorgilakis) [reply CHANGELOG.md:8]: This exists. So remove lines 7-8?

Comment (NicolasLiampotis) [reply CHANGELOG.md:8]: You're right - then let's remove lines 7-8

**Comment by cgeorgilakis** [comment]
Requested changes have been implemented.

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**